WebRoot/skins/_base/base4/skin.properties

```diff
@@ -564,6 +567,47 @@ ButtonLeftIcon                  = margin: 0; text-align:center;
 ButtonRightIcon                 = margin: 0; text-align:center;
 ButtonDropDown                  = margin: 0; text-align:center;
 
+#Alt button
+AltButton                          = @Button@
+AltButtonBg                        = #a8a8a8
+AltButtonActiveBg                  = @darken(AltButtonBg,25)@
+AltButtonDisabledBg                = #d8d8d8
+AltBtnTextColor                    = @white@
+AltBtnDisabledTextColor            = #666666
+
+AltButtonBorder                    = padding:0 12px; border:none; @MediumRoundCorners@;
+AltButtonBorder-normal             = background:@AltButtonBg@; box-shadow:0 0 1px 1px rgba(0,0,0,0.2);
+AltButtonBorder-disabled           = background:@AltButtonDisabledBg@; box-shadow:none; opacity:0.6;
+AltButtonBorder-hover              = background:@AltButtonActiveBg@
+AltButtonBorder-active             = @AltButtonBorder-hover@
+AltButtonBorder-focused            = @AltButtonBorder-hover@
+AltButtonBorder-selected           = @AltButtonBorder-hover@
+
+AltButtonText                      = color:@AltBtnTextColor@; text-transform:uppercase; font-weight:300; @FontSize-normal@;
```

Comment:
do we need height/width as auto here? isn't inline-block will take it's height/width as per content size?